livekit-rtc/livekit/rtc/audio_filter.py

```diff
@@ -0,0 +1,32 @@
+import json
+from typing import Optional
+
+from ._ffi_client import FfiClient, FfiHandle
+from ._proto import ffi_pb2 as proto_ffi
+
+
+class AudioFilter:
+    def __init__(
+        self, path: str, url: str, token: str, dependencies: Optional[list[str]] = None
+    ) -> None:
+        self._path = path
+
+        req = proto_ffi.FfiRequest()
+        req.load_audio_filter_plugin.plugin_path = path
+        req.load_audio_filter_plugin.dependencies[:] = (
+            dependencies if dependencies is not None else []
+        )
+        req.load_audio_filter_plugin.options = json.dumps(
+            {
+                "url": url,
+                "token": token,
+            }
```

Comment:
Should we make it strictly typed inside our protobufs definition instead of using json? The auth function could accept those as arguments (no need for a prost dependency)